src/lib/CLValue2/String.ts

```diff
@@ -0,0 +1,33 @@
+import { CLType, CLValue } from "./Abstract"; 
+
+export class CLStringType extends CLType {
+  toString(): string {
+    return 'String';
+  }
+}
+
+export class CLString extends CLValue {
```

Comment:
I like `CLString` naming concept. Maybe all types should follow that?

Reply:
Yeap, seems to be better convention than `MapValue` for example